# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/chrome-webidl/WindowGlobalActors.webidl
# Commit: 405100db6c45
# Full Hash: 405100db6c45f0463286067df5b491f74790d9aa
# Author: Kashav Madan <kmadan@mozilla.com>
# Date: 2019-07-11 09:51:12
# Regressor Bug: 1543251
# File Overlap Count: 1
# Description:
#   Bug 1543251 - Move hasBeforeUnload from PBrowser to PWindowGlobal, r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D37003
# ==============================================================================

diff -r 0c408343080d -r 405100db6c45 dom/chrome-webidl/WindowGlobalActors.webidl
--- a/dom/chrome-webidl/WindowGlobalActors.webidl	Thu Jul 04 07:41:57 2019 +0000
+++ b/dom/chrome-webidl/WindowGlobalActors.webidl	Wed Jul 10 21:13:44 2019 +0000
@@ -27,6 +27,9 @@
   // embedder is in a different process.
   readonly attribute boolean isProcessRoot;
 
+  // True if this window has registered a "beforeunload" event handler.
+  readonly attribute boolean hasBeforeUnload;
+
   // Is the document loaded in this WindowGlobalParent the initial document
   // implicitly created while "creating a new browsing context".
   // https://html.spec.whatwg.org/multipage/browsers.html#creating-a-new-browsing-context